project/plugins.sbt

```
@@ -2,6 +2,9 @@ val scalaJSVersion =
   Option(System.getenv("SCALAJS_VERSION")).filter(_ != "").getOrElse("1.0.0")
 
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % scalaJSVersion)
-addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "0.6.0")
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.0.0")
+
+addSbtPlugin("org.scala-native"   % "sbt-scala-native" % "0.4.0-SNAPSHOT")
```

It can now be updated :-)
```suggestion
addSbtPlugin("org.scala-native"   % "sbt-scala-native" % "0.4.0")
```